### DISCUSSION TITLE
OSError: [WinError 6] Invalid driver

### DISCUSSION THREAD

**Discussion by Alejogb1** [ultrafunkamsterdam/undetected-chromedriver#1385] [General]
got this while running python scrape_products.py

**Comment by nullANzero** [comment]
I don't want to be rude, however read the documentation + look in open/closed Issues maybe there is already a solution there... Since a lot of people write about this error that occurs when closing
Solution: https://github.com/ultrafunkamsterdam/undetected-chromedriver/pull/1256
